### PR TITLE
PHOENIX-6296 Synchronize @Parameters, @BeforeClass and @AfterClass me…

### PR DESCRIPTION
**Pull request by stoty**
…thods take 2

### CODE REVIEW

**Review by virajjasani — APPROVED** [review]
+1 (non-binding), pending QA

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewIndexIdRetrieveIT.java [hunk 1]
```diff
@@ -62,7 +62,7 @@
             " FROM SYSTEM.CATALOG WHERE TABLE_NAME='%s' AND COLUMN_COUNT IS NOT NULL";
 
     @BeforeClass
-    public static void setUp() throws Exception {
+    public static snychronized void setUp() throws Exception {
```
Comment (gjacoby126) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewIndexIdRetrieveIT.java:65]: typo here: shouldn't compile

**Review by gjacoby126 — CHANGES_REQUESTED** [review]
Typos in synchronized keyword

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java [hunk 2]
```diff
@@ -76,7 +76,7 @@
     private static Connection conn = null;
 
     @BeforeClass
-    public static void init() throws SQLException {
+    public static synchornized void init() throws SQLException {
```
[the remark this answers is not in the document]
Comment (stoty) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java:79]: It's not, I was wrongly relying on the automated tests. 
Thanks for niticing it.

**Comment by stoty** [comment]
Pushed new version with fixed typos.

**Review by gjacoby126 — APPROVED** [review]
+1, thanks for the fix

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 31s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 30s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  10m  1s |  master passed  |
| +1 :green_heart: |  compile  |   1m 20s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 34s |  master passed  |
| +1 :green_heart: |  javadoc  |   1m  2s |  master passed  |
| +0 :ok: |  spotbugs  |   2m 57s |  phoenix-core in master has 973 extant spotbugs warnings.  |
| +0 :ok: |  spotbugs  |   0m 37s |  phoenix-tools in master has 3 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| -1 :x: |  mvninstall  |   0m 52s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 43s |  phoenix-core in the patch failed.  |
| -1 :x: |  javac  |   0m 43s |  phoenix-core in the patch failed.  |
| +1 :green_heart: |  checkstyle  |   0m 34s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   0m 59s |  the patch passed  |
| -1 :x: |  spotbugs  |   0m 40s |  phoenix-core in the patch failed.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m 41s |  phoenix-core in the patch failed.  |
| +1 :green_heart: |  unit  |   2m 50s |  phoenix-tools in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 19s |  The patch does not generate ASF License warnings.  |
|  |   |  32m 39s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/1055 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs hbaseanti checkstyle compile |
| uname | Linux 7960c2b89c78 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / 4d5449f |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| mvninstall | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/patch-mvninstall-root.txt |
| compile | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/patch-compile-phoenix-core.txt |
| javac | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/patch-compile-phoenix-core.txt |
| spotbugs | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/patch-spotbugs-phoenix-core.txt |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/artifact/yetus-general-check/output/patch-unit-phoenix-core.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/testReport/ |
| Max. process+thread count | 579 (vs. ulimit of 30000) |
| modules | C: phoenix-core phoenix-tools U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/1/console |
| versions | git=2.7.4 maven=3.3.9 spotbugs=4.1.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 31s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   5m 27s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  10m 11s |  master passed  |
| +1 :green_heart: |  compile  |   1m 17s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 34s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 59s |  master passed  |
| +0 :ok: |  spotbugs  |   2m 56s |  phoenix-core in master has 973 extant spotbugs warnings.  |
| +0 :ok: |  spotbugs  |   0m 36s |  phoenix-tools in master has 3 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   7m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 17s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 17s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 35s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   3m 55s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 106m 59s |  phoenix-core in the patch failed.  |
| +1 :green_heart: |  unit  |   3m 12s |  phoenix-tools in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 50s |  The patch does not generate ASF License warnings.  |
|  |   | 151m  3s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/1055 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs hbaseanti checkstyle compile |
| uname | Linux c6b154da8b14 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / 4d5449f |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/2/artifact/yetus-general-check/output/patch-unit-phoenix-core.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/2/testReport/ |
| Max. process+thread count | 6729 (vs. ulimit of 30000) |
| modules | C: phoenix-core phoenix-tools U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1055/2/console |
| versions | git=2.7.4 maven=3.3.9 spotbugs=4.1.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request closed without merging**